lib/licensee/matchers/spdx.rb

```diff
@@ -0,0 +1,16 @@
+module Licensee
+  module Matchers
+    class Spdx < Licensee::Matchers::Package
+      # While we could parse the LICENSE.spdx file, prefer
+      # a lenient regex for speed and security. Moar parsing moar problems.
+      LICENSE_REGEX = /PackageLicenseDeclared:\s*([a-z\-0-9\. +()]+)\s*/i
```

Comment:
This will work in single-line cases, which are the most likely.  But the current tag/value format used in `.spdx` is not very clear about values wrapping over multiple lines (spdx/spdx-spec#44).  You may want a balanced-paren parser (reading to the end of the line or until any parens opened in that line are closed).  Or you can punt until the SPDX spec expresses a clearer opinion on this.

Reply:
Thanks for mentioning this. I think it makes sense to punt until there's a clear opinion. I subscribed to the issue.